activesupport/test/core_ext/duration_test.rb

```diff
@@ -653,7 +654,7 @@ def test_iso8601_output_precision
   def test_iso8601_output_and_reparsing
     patterns = %w[
       P1Y P0.5Y P0,5Y P1Y1M P1Y0.5M P1Y0,5M P1Y1M1D P1Y1M0.5D P1Y1M0,5D P1Y1M1DT1H P1Y1M1DT0.5H P1Y1M1DT0,5H P1W +P1Y -P1Y P-1Y
-      P1Y1M1DT1H1M P1Y1M1DT1H0.5M P1Y1M1DT1H0,5M P1Y1M1DT1H1M1S P1Y1M1DT1H1M1.0S P1Y1M1DT1H1M1,0S P-1Y-2M3DT-4H-5M-6S
+      P1Y1M1DT1H1M P1Y1M1DT1H0.5M P1Y1M1DT1H0,5M P1Y1M1DT1H1M1S P1Y1M1DT1H1M1.0S P1Y1M1DT1H1M1,0S P-1Y-2M3DT-4H-5M-6S PT1000000.00S
```

Comment:
Is the precision 2 on this test, and that's why we get that string?

Is there already a test for `sprintf("%0.0#{@precision}f", seconds)`?

Reply:
Ah, thanks! this won't get generated anymore and there are other tests here for different second precisions. I'll remove this change.